django/utils/version.py

```diff
@@ -77,3 +78,14 @@ def get_git_changeset():
     except ValueError:
         return None
     return timestamp.strftime('%Y%m%d%H%M%S')
+
+
+def get_version_tuple(version):
+    """Return tuple version (X, Y, Z) from VERSION if present."""
```

Comment:
`VERSION` is capitalized in all other doc strings from this file. So, I thought it should be.